src/main/resources/default/taglib/t/blobImageHardRefField.html.pasta

```diff
@@ -171,4 +200,36 @@
             outerDiv.querySelector('#saveHint').classList.remove('d-none');
         }
     }
+
+    function isDisplayableImage(extension) {
+        return extension === '.jpg' || extension === '.jpeg' || extension === '.png';
```

Comment:
webp noch?

Reply:
will follow with an extra PR in the next weeks.